### ISSUE TITLE
Cannot set property 'required' of undefined in card.js on copyProps

### ISSUE THREAD

**Issue by whoisme555** [bootstrap-vue/bootstrap-vue#5464]
I'm sure there's something in my solution that's breaking this, but I'm not sure what it could be. When importing bootstrap-vue with the following code:

``` 
    import Vue from "vue";
    import BootstrapVue from "bootstrap-vue";
    Vue.use(BootstrapVue); 
```

I'm getting the error in the title in card.js on line 15. An inspection shows copyProps setting what should be imgSrc to imgUndefined and I have no idea why. Anyone know?

**Comment by whoisme555** [comment]
I figured it out. Since this is an old application I'm trying to slowly migrate, we're using Mootools. And Mootools has a bind override that's causing conflicts. Not sure how to fix that yet

**Comment by whoisme555** [comment]
In case any of the other 7 people stuck using mootools run into this problem - The fix for me was dealing with the 1.2 compatibility bind override. For whatever reason, mootools likes playing with prototypes, and the bind was taking args || arguments. It needed to combine the two (or just use the non compatibility bind override)
